### ISSUE TITLE
AMP

### ISSUE THREAD

**Issue by osentenac** [soflyy/oxygen-bugs-and-features#2125]
Features to setup AMP with Articles, Pagaes, Templates,...

**Comment by ymarketing** [comment]
Yes, having compatibility with the official AMP plugin would be a benefit. https://wordpress.org/plugins/amp/
